Riot/Modules/Common/Recents/Service/MatrixSDK/RecentsListService.swift

```diff
@@ -651,8 +794,10 @@ private struct FetcherTypes: OptionSet {
     static let lowPriority = FetcherTypes(rawValue: 1 << 6)
     static let serverNotice = FetcherTypes(rawValue: 1 << 7)
     static let suggested = FetcherTypes(rawValue: 1 << 8)
+    static let recents = FetcherTypes(rawValue: 1 << 9)
```

Comment:
Does this one need updating to to `breadcrumbs` too?

Reply:
Sorry for having missed all of these ...

Reply:
Oh no, no need to apologise 🙂